### ISSUE TITLE
IPv4CIDRToRange not found

### ISSUE THREAD

**Issue by clarsen** [ClickHouse/ClickHouse#5703]
**Describe the bug**

Maybe some functionality is disabled in the config -- I didn't make changes from default other than to enforce network ACLs, and bump up max_memory_usage, but while the sources for 19.5.3.8 do show the function is registered, the server doesn't recognize the function.

**How to reproduce**
$ lsb_release -a
```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic
```

$ sudo dpkg -l 'clickhouse-server*'
```
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                                        Version                    Architecture               Description
+++-===========================================-==========================-==========================-===========================================================================================
ii  clickhouse-server                           19.5.3.8                   all                        Server binary for ClickHouse
un  clickhouse-server-base                      <none>                     <none>                     (no description available)
un  clickhouse-server-common                    <none>                     <none>                     (no description available)
```

$ clickhouse-client -V
```
ClickHouse client version 19.5.3.8 (official build).
```

`SELECT IPv4CIDRToRange(toIPv4('192.168.5.2'), 0)`  gives:
```
Code: 46. DB::Exception: Received from 127.0.0.1:9440. DB::Exception: Unknown function IPv4CIDRToRange.
```

**Expected behavior**
Would expect it to not error and return a UInt32 tuple instead.

**Error message and/or stacktrace**

$ clickhouse-client -m -A --port 9440 -h 127.0.0.1 -s
```
ClickHouse client version 19.5.3.8 (official build).
Connecting to 127.0.0.1:9440 as user default.
Connected to ClickHouse server version 19.5.3 revision 54417.

SERVERHOST :) SELECT IPv4CIDRToRange(toIPv4('192.168.5.2'), 0);

SELECT IPv4CIDRToRange(toIPv4('192.168.5.2'), 0)

Received exception from server (version 19.5.3):
Code: 46. DB::Exception: Received from 127.0.0.1:9440. DB::Exception: Unknown function IPv4CIDRToRange.

0 rows in set. Elapsed: 0.003 sec.
```

**Additional context**

**Comment by 4ertus2** [comment]
I've looked in code. There's IPv4CIDRToRange function there (with upper cased T).
There's an error in docs. Russian version also contains wrong name IPv4CIDRtoIPv4Range.

**Comment by BayoNet** [comment]
En docs were fixed 7 days ago in the https://github.com/yandex/ClickHouse/commit/4f408cef1281b7eb86b6be3edc9322c1d4e18227#diff-5a70d14804c5e8dd00569e461bdca911 commit.
Ru docs I fixed right now in the #5706.

Examples from docs should work:

```
$ clickhouse-client 
ClickHouse client version 19.10.1.683 (official build).
Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 19.10.1 revision 54422.

bayonet-VirtualBox :) SELECT IPv4CIDRToRange(toIPv4('192.168.5.2'), 16)

SELECT IPv4CIDRToRange(toIPv4('192.168.5.2'), 16)

┌─IPv4CIDRToRange(toIPv4('192.168.5.2'), 16)─┐
│ ('192.168.0.0','192.168.255.255')          │
└────────────────────────────────────────────┘

1 rows in set. Elapsed: 0.023 sec. 
```
